Makefile

```diff
@@ -4,7 +4,7 @@ CPPFLAGS += -DVERSION=\"$(VERSION)\" -D_POSIX_C_SOURCE=199309L -DYA_INTERNAL -DY
 			-DYA_BSPWM
 CFLAGS += -std=c99 -Iinclude -pedantic -Wall -flto -O2 `pkg-config --cflags pango pangocairo libconfig gdk-pixbuf-2.0 alsa`
 
-DEPS += pango pangocairo libconfig gdk-pixbuf-2.0 alsa
+DEPS += pango pangocairo libconfig gdk-pixbuf-2.0 alsa xkbcommon xkbcommon-x11 xcb-xkb
```

Comment:
These can probably be included with `LDLIBS` (`-lxkbcommon -lxkbcommon-x11 -lxcb-xkb`).